src/Umbraco.Web.UI/Umbraco/PartialViewMacros/Templates/EditProfile.cshtml

```diff
@@ -10,15 +9,19 @@
 
     Html.EnableClientValidation();
     Html.EnableUnobtrusiveJavaScript();
-    Html.RequiresJs("https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js");
-    Html.RequiresJs("https://cdnjs.cloudflare.com/ajax/libs/jquery-validate/1.19.0/jquery.validate.min.js");
-    Html.RequiresJs("https://cdnjs.cloudflare.com/ajax/libs/jquery-validation-unobtrusive/3.2.11/jquery.validate.unobtrusive.min.js");
+
+    var runtimeMinifier = Current.RuntimeMinifier;
+
+    runtimeMinifier.RequiresJs("umbraco",
```

Comment:
as mentioned in my previous comment for these macro scripts we shouldn't use the minifier to register scripts, the user should choose if they want to use a runtime minifier. Also because this won't work OOTB if there are more than one of these on the page or another cshtml that has RenderJsHere which forces users to update all these macros anyways. We should just render normal script tags and keep it simple. i'll update. (plus like mentioned before this wont work with load balancing)